client/src/App.js

```diff
@@ -173,7 +209,38 @@ class App extends Component {
         </div>
         <Instructions
           ganacheAccounts={this.state.ganacheAccounts}
-          name={instructionsKey} accounts={this.state.accounts} />
+          name={instructionsKey} accounts={this.state.accounts}/>
+      </div>
+    );
+  }
+
+  renderGaslessBody() {
+    return (
+      <div className={styles.wrapper}>
+        {!this.state.web3 && this.renderLoader()}
+        {this.state.web3 && !this.state.gaslessContract && (
+          this.renderDeployCheck('gasless-counter')
+        )}
+        {this.state.web3 && this.state.gaslessContract && (
+          <div className={styles.contracts}>
+            <h1>Gasless Counter Contract</h1>
+            <p>In order to make gasless transactions, press the 'Use Relayer' button below. </p>
+            <p>(to stop using relayer simply refresh the page)</p>
+            <Button onClick={() => useRelayer(this.state.web3)}>
+              Use Relayer
```

Comment:
The button provides no feedback to the user. Maybe we could change it to a toggle, or provide any kind of visual cue that the relayer is in use? It's hard to know whether the click worked or not.

Reply:
I will change this to a toggle able button

Reply:
When user clicks button, it becomes disabled and the text changes to 'Using Relayer'